hydpy/models/rconc_nash.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by linear storage cascade model ('Nash cascade').
+
+|rconc_nash| is a submodel that supplies its main model with the
+calculation of the runoff concentration by the storage cascade approach.
+
+The integration test of the application model |hland_v3| uses the
+|rconc_nash| submodel.
```

Comment:
See the integration tests of the application model "h3", which use "r" as a submodel.

Reply:
ok